packages/vm/lib/exceptions.ts

```diff
@@ -16,6 +16,7 @@ export enum ERROR {
   INVALID_BEGINSUB = 'invalid BEGINSUB',
   INVALID_RETURNSUB = 'invalid RETURNSUB',
   INVALID_JUMPSUB = 'invalid JUMPSUB',
+  EIP3541_VIOLATION = 'eip3541 violation: cannot put contracts starting with 0xEF bytecode',
```

Comment:
Maybe we can use something more semantically expressive here - a bit analog to the last discussions we had coming to the conclusion to rather avoid EIP numbers for names? 🤔

Suggestion (not sure yet though):

- `RESTRICTED_STARTING_BYTECODE`

Or similar.

Reply:
(same a bit for the function name from above)

Reply:
In some place seen `INVALID_CODE` or `INVALID_BYTECODE`, and in EVMC we also had `CONTRACT_VALIDATION_FAILED` from earlier, that may be also a good one in the future.

Reply:
Ah yeah, that's better, my suggestion is likely still to specific, I like `INVALID_BYTECODE`. 😄

Reply:
But `CONTRACT_VALIDATION_FAILED` might be a bit better targeted, so eventually more suited.

Reply:
Probably `INVALID_CODE`/`INVALID_BYTECODE` seems to be the best for now, and if any "real validation" is rolled out later (such as EIP-3540), then the `CONTRACT_VALIDATION_FAILED` error could be introduced in addition.